test/inductor/test_torchinductor.py

```diff
@@ -13026,9 +13029,11 @@ def fn3(x):
                 ),
                 (
                     fn3,
-                    "triton_poi_fused_layer_norm_relu"
-                    if torch._dynamo.config.inline_inbuilt_nn_modules
-                    else "triton_poi_fused_LayerNorm_ReLU",
+                    (
```

Comment:
Is there any reason for the changes to this file? They seem unintentional and mess up the blame

Reply:
I think it was just the linter jumping in and auto-fixing it. I'll remove them.